### PR TITLE
Ceilometer Support

### PR DESCRIPTION
**Pull request by avibh**
Ceilometer support for 1.8.x
work in process since there is a real PR pending JClouds review. this is so insights team can start working with ceilometer api's

### CODE REVIEW

**Comment by avibh** [comment]
changes applied

File: apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/MetersApi.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import java.util.List;
+
+
+/**
+ * Provides access to Meters features.
+ *
+ */
+@RequestFilters(AuthenticateRequest.class)
+@Consumes(MediaType.APPLICATION_JSON)
+@Path("/v2/meters")
```
Comment (cloudband) [inline apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/MetersApi.java:44]: Don't hardcode the version in the API path (it's not done for other services as well)

File: apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/QueryApi.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import java.util.List;
+
+
+/**
+ * Provides access to Meters features.
+ *
+ */
+@RequestFilters(AuthenticateRequest.class)
+@Consumes(MediaType.APPLICATION_JSON)
+@Path("/v2/query/")
```
Comment (cloudband) [inline apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/QueryApi.java:43]: Don't hardcode the version in the API path (it's not done for other services as well)

File: apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/MetersApi.java (hunk 1 above)
Comment (avibh) [inline apis/openstack-ceilometer/src/main/java/org/jclouds/openstack/ceilometer/v2/features/MetersApi.java:44]: i've raised this issue in the real PR, let's leave it for them to resolve

**Pull request merged**